kimchi/src/tests/foreign_field_add.rs

```diff
@@ -29,9 +30,10 @@ use num_bigint::{BigUint, RandBigInt};
 use num_traits::FromPrimitive;
 use o1_utils::{
     foreign_field::{ForeignElement, HI, LO, MI, TWO_TO_LIMB},
-    FieldHelpers,
+    FieldHelpers, Two,
```

Comment:
I think `Two` may have got dragged in by some other merge/branch that we don't want?

Reply:
Oh, the Two still exists, but not used everywhere as proposed in the other PR

Reply:
It started being used partly in Rot a few weeks ago, but then the PR to use it everywhere else in the code was not accepted. I wonder if others may want to remove it completely, but it was really helpful for certain Keccak gates at least for me.